test/EFCore.MySql.IntegrationTests/README.md

```diff
@@ -38,7 +40,7 @@ Methods:
 }
 ```
 
-The `scripts` directory contains load testing scripts. These scripts require that the  [Vegeta](https://github.com/tsenart/vegeta/releases) binary is installed and accessible in your PATH. Here are examples of how to call the load testing scripts:
+The `scripts` directory contains load testing scripts. These scripts require that the  [Vegeta](https://github.com/tsenart/vegeta/releases) binary is installed and accessible in your PATH. Here are some examples of how to call the load testing scripts:
 ```
 # by default, runs 50 async queries per second for 5 seconds
 ./stress.sh     # bash for linux
```

Comment:
While you are at it, please remove the `stress.sh` script. The `stress.ps1` script should work on Linux as well.

These lines can then be changed to:

> The `scripts` directory contains a load testing script `stress.ps1`. This script requires that the [Vegeta](https://github.com/tsenart/vegeta/releases) binary is installed and accessible in your PATH environment variable. Here are some examples of how to call the script:
>
>
> ```powershell
> # runs 50 async queries per second for 5 seconds by default 
> ./stress.ps1
> 
> # run 100 sync queries per second for 1 minute
> ./stress.ps1 100 1m sync
> ```

Reply:
Like in [comment](https://github.com/PomeloFoundation/Pomelo.EntityFrameworkCore.MySql/pull/1872#discussion_r1520363192) wil make this `runs` ?

Reply:
Sure. Any of those two is fine. Whatever is consistent with the other comments is best.

Reply:
> While you are at it, please remove the `stress.sh` script. The `stress.ps1` script should work on Linux as well.

@ragavendra Once `stress.sh` has been removed, we can merge this PR.

Reply:
> > While you are at it, please remove the `stress.sh` script. The `stress.ps1` script should work on Linux as well.
> 
> @ragavendra Once `stress.sh` has been removed, we can merge this PR.

@ragavendra Any update on this?

Reply:
@lauxjpn ... Please check now. Sorry I was working on my project stopsnearme.ca for a while now. Please keep me posted if any help is needed.